README.md

```diff
@@ -196,6 +196,49 @@ using (var provider = services.BuildServiceProvider()) {
 }
 ```
 
+# Using the installer template
+
+The installer template is mostly configured from the command live and the config .json file, but more advanced changes may require modifying the project files. Feel free to copy the installer project and modifying it to your needs. The following instructions produce a single file executable for a .NET core project. It assumes that you have a config file named `config.example.yml` somewhere.
+
+ - Update the `setup-config.json` file to suit your needs. The default fields are described below. All config fields except for `setup_project` is injected as a build property with msbuild into the `.wixproj` file.
+ - Modify the `InstBanner.bmp` file in `Resources`, adding your extractor name to the blue field.
+ - Modify or replace the `License.rtf` with a license adapted to your project.
+ - (Optional) Add any new additional config files at the bottom of the `Product.wxs` file, by adding new `Component` blocks after the one for `config.example.yml`
+ - (Optional) Add a cognite icon to your extractor by adding `Resources/black32x32.ico` to the folder for your actual extractor, and adding `<ApplicationIcon>black32x32.ico</ApplicationIcon>` to its .csproj file.
+ - Compile the installer in a windows environment with Wix utils and msbuild installed, using something like the following command:
+ 
+`build.ps1 -b Path\To\MSBuild.exe -v 1.0.0 -d "some description" -c Path\To\setup-config.json`
```

Comment:
Do you know if this is something that can be done from GitHub actions? I guess if there is an action template with wix tools installed, it should work

Reply:
You'll need wix toolset, yeah. If that can somehow be installed automatically you're good. You don't need wix 4, I've used 3.x, which has been around for some time, so I think there's a decent chance.